src/Provider/Provider.Web/Configuration/ConfigurationExtensions.cs

```diff
@@ -71,10 +72,14 @@ public static void AddMvcService(this IServiceCollection services, IHostingEnvir
                 opts.Filters.AddService<PlannedOutageResultFilter>();
 
                 opts.AddTrimModelBinderProvider(loggerFactory);
+                if (EnvironmentNames.GetTestEnvironmentNames().Contains(hostingEnvironment.EnvironmentName.ToUpper()) == false)
```

Comment:
The if condition can be converted into meaningful function with name like "IsProductionEquivalentEnvironment"

Reply:
It's the same functionality used for Employer... ! i will give it a meaningful name.